src/app/app-platform/ContentAppPlatform.cpp

```diff
@@ -455,6 +455,61 @@ uint32_t ContentAppPlatform::GetPincodeFromContentApp(uint16_t vendorId, uint16_
     return (uint32_t) strtol(pinString.c_str(), &eptr, 10);
 }
 
+// Returns ACL entry with match subject or CHIP_ERROR_NOT_FOUND if no match is found
+CHIP_ERROR ContentAppPlatform::GetACLEntryIndex(size_t * foundIndex, FabricIndex fabricIndex, NodeId subjectNodeId)
```

Comment:
What is this used for?

Reply:
Used in the next method to find any existing ACL for the given subject. This is the app platform code using for casting and the ACL entries created follow the device library spec (casting video player section).